src/includes/header.php

```diff
@@ -13,17 +13,27 @@
     <link rel="shortcut icon" href="/static/images/favicon.ico?<?php echo $assetVersion ?>" type="image/x-icon">
     <link rel="stylesheet" href="/static/dist/all.min.css?<?php echo $assetVersion ?>">
     <script src="/static/dist/all.min.js?<?php echo $assetVersion ?>"></script>
+    <script src="/static/js/cookieBanner.js?<?php echo $assetVersion ?>"></script>
```

Comment:
@walsh-conor I just noticed this command in the make file.

https://github.com/Learnosity/learnosity-demos/blob/develop/Makefile#L29

Have you tried adding the cookie banner to this array:

https://github.com/Learnosity/learnosity-demos/blob/develop/gulpfile.js#L8

then run `make update-assets`

It looks like that runs gulp and should compile the cookie banner file into `all.min.js`